worlds/smz3/TotalSMZ3/Patch.py

```diff
@@ -351,6 +351,23 @@ def GetZ3ItemId(self, location: Location):
                 not (item.IsDungeonItem() and location.Region.IsRegionItem(item) and item.World == self.myWorld) else itemDungeon
             
             return value.value
+        elif (location.APLocation.item.game == "A Link to the Past"):
+            if location.APLocation.item.code + 84000 in lookup_id_to_name:
+                return location.APLocation.item.code
```

Comment:
This bit is required to prevent this issue (https://discord.com/channels/731205301247803413/959203096142184458/1085919074749513819). In the room that breaks, one of the item is "Bee" from ALTTP which maps to an invalid graphic SMZ3 item in the ROM. I verified that it fixes the issue.
```suggestion
            if location.APLocation.item.code + 84000 in lookup_id_to_name:
                ALTTPBottleContentCodeToSMZ3ItemCode = {
                    ItemType.RedContent.value: ItemType.BottleWithRedPotion.value,
                    ItemType.GreenContent.value: ItemType.BottleWithGreenPotion.value,
                    ItemType.BlueContent.value: ItemType.BottleWithBluePotion.value,
                    ItemType.BeeContent.value: ItemType.BottleWithBee.value,
                }
                return ALTTPBottleContentCodeToSMZ3ItemCode.get(location.APLocation.item.code, location.APLocation.item.code)
            else:
                return ItemType.Something.value
```

Reply:
Yeah I saw that there was a bee item in the room and figured that was the issue and planned to just make them show as an AP icon but this works too I guess. Thank you